docs: improve README examples for `stats/base/dists/t` namespace

Jai0401 opened the pull request:
Resolves #1645 

## Description

> What is the purpose of this pull request?

This pull request:
Improve README examples of `stats/base/dists/t` namespace

## Related Issues

> Does this pull request have any related issues?
No.

This pull request:

-   fixes #1645 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

Jai0401 committed:
docs:improve README examples of  namespace

Jai0401 committed:
docs:improve README examples of t namespace

kgryte commented on lib/node_modules/@stdlib/stats/base/dists/t/README.md:
```diff
@@ -112,6 +112,23 @@ var objectKeys = require( '@stdlib/utils/keys' );
 var t = require( '@stdlib/stats/base/dists/t' );
 
 console.log( objectKeys( t ) );
+
```
```suggestion
```

kgryte replied:
You should remove the `utils/keys` import.

kgryte commented on lib/node_modules/@stdlib/stats/base/dists/t/examples/index.js:
```diff
@@ -22,3 +22,20 @@ var objectKeys = require( '@stdlib/utils/keys' );
 var t = require( './../lib' );
 
 console.log( objectKeys( t ) );
+
```
```suggestion
```

kgryte commented on lib/node_modules/@stdlib/stats/base/dists/t/README.md:
```diff
@@ -112,6 +112,23 @@ var objectKeys = require( '@stdlib/utils/keys' );
 console.log( objectKeys( t ) );
+
+// Example usage of specific functions from the t namespace
+var degreesOfFreedom = 3;
+var xValue = 1.5;
+
+// Evaluate the probability density function (PDF) at a specific value
+var pdfResult = t.pdf( xValue, degreesOfFreedom );
+console.log( 'PDF at x = ' + xValue + ': ' + pdfResult );
+
+// Evaluate the cumulative distribution function (CDF) at a specific value
+var cdfResult = t.cdf( xValue, degreesOfFreedom );
+console.log( 'CDF at x = ' + xValue + ': ' + cdfResult );
+
+// Get the mean and variance of the t distribution
+var meanValue = t.mean( degreesOfFreedom );
+var varianceValue = t.variance( degreesOfFreedom );
+console.log( 'Mean: ' + meanValue + ', Variance: ' + varianceValue );
```
```suggestion
var dof = 3;
var x = 1.5;

// Evaluate the probability density function (PDF) at a specific value:
var res = t.pdf( x, dof );
console.log( 'PDF at x = ' + x + ': ' + res );

// Evaluate the cumulative distribution function (CDF) at a specific value:
res = t.cdf( x, dof );
console.log( 'CDF at x = ' + x + ': ' + res );

// Get the mean and variance of the t distribution:
var mu = t.mean( dof );
var v = t.variance( dof );
console.log( 'Mean: ' + mu + ', Variance: ' + v );
```

Follow project conventions in preferring shorter variable names.

Jai0401 committed:
chore: done-required-changes-for-improving-readme-examples-in-t-namespace

Jai0401 commented:
@kgryte Thank you for your feedback! I have addressed all the issues you mentioned in your review. Please take a look at the changes, and let me know if everything looks good now. If you have any further suggestions or concerns, feel free to share them. Thanks again!

Planeshifter reviewed (approved):
Looks alright; let's land this change. Thanks!
